starcheck/star_probs.py

```diff
@@ -0,0 +1,221 @@
+"""
+Functions related to probabilities for star acquisition and guide tracking.
+"""
+
+from __future__ import print_function, division
+
+from numba import jit
+from itertools import izip
+
+from scipy.optimize import brentq
+import numpy as np
+from Chandra.Time import DateTime
+
+# Scale and offset fit of polynomial to acq failures in log space.
+# Derived in the fit_sota_model_probit.ipynb IPython notebook for data
+# covering 2007-Jan-01 - 2015-July-01.  This is in the state_of_aca repo.
+#
+# scale = scl2 * m10**2 + scl1 * m10 + scl0, where m10 = mag - 10,
+# and likewise for offset.
+
+
+WARM_THRESHOLD = 100  # Value (N100) used for fitting
+
+SOTA_FIT_NO_1P5 = [9.6887121605441173,  # scl0
+                   9.1613040261776177,  # scl1
+                   -0.41919343599067715,  # scl2
+                   -2.3829996965532048,  # off0
+                   0.54998934814773903,  # off1
+                   0.47839260691599156]  # off2
+
+SOTA_FIT_ONLY_1P5 = [8.541709287866361,
+                     0.44482688155644085,
+                     -3.5137852251178465,
+                     -1.3505424393223699,
+                     1.5278061271148755,
+                     0.30973569068842272]
+
+
+def t_ccd_warm_limit(date, mags, colors=0,
+                     min_n_acq=5.0,
```

Comment:
Do you have a plan to update this to use a probability value instead of min_n_acq to determine warm limit?  I'd like to use a version with that in the LTS checker soon if that is the way we want to go in starcheck.